pkgs/development/python-modules/pyliblo/default.nix

```diff
@@ -23,6 +23,11 @@ buildPythonPackage rec {
       url = "https://git.alpinelinux.org/aports/plain/community/py3-pyliblo/py3.11.patch?id=a7e1eca5533657ddd7e37c43e67e8126e3447258";
       hash = "sha256-4yCWNQaE/9FHGTVuvNEimBNuViWZ9aSJMcpTOP0fnM0=";
     })
+    # Fix compile error due to  incompatible pointer type 'lo_blob_dataptr'
+    (fetchurl {
+      url = "https://github.com/s0600204/pyliblo/commit/ebbb255d6a73384ec2560047eab236660d4589db.patch";
```

Comment:
Usually we don't fetch commits from forks, since they might disappear on force push. The patch is quite small so we can just vendor it.

Reply:
I've use the url to the commit within the main repo

Reply:
That has no effect, in fact you can put any username there that has a fork of the repository.

Reply:
If the fork is deleted, the commit will still stay in the upstream repo.

Reply:
Alright, I have done some testing. Here is a commit from a [deleted fork (web archive)](https://web.archive.org/web/20250104215502/https://github.com/FliegendeWurst/cpr/commit/77fe6031188cf054aba791608c1a8db19c1a24fa).

It seems that the commit stays around https://github.com/libcpr/cpr/commit/77fe6031188cf054aba791608c1a8db19c1a24fa